NullReferenceException in RemoveDuplicatesWithLastOneWinsPolicy.Execute

Keep seeing this when trying to run `System.Linq.Expressions` unit tests in `corefx` in VS 2015:

```
1>------ Build started: Project: System.Linq.Expressions.Tests, Configuration: Debug Any CPU ------
1>  System.Linq.Expressions.Tests -> C:\Projects\bartdesmet-corefx\corefx\bin\AnyOS.AnyCPU.Debug\System.Linq.Expressions.Tests\System.Linq.Expressions.Tests.dll
1>C:\Projects\bartdesmet-corefx\corefx\Tools\publishtest.targets(97,5): error MSB4018: The "RemoveDuplicatesWithLastOneWinsPolicy" task failed unexpectedly.
1>C:\Projects\bartdesmet-corefx\corefx\Tools\publishtest.targets(97,5): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object.
1>C:\Projects\bartdesmet-corefx\corefx\Tools\publishtest.targets(97,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.RemoveDuplicatesWithLastOneWinsPolicy.Execute()
1>C:\Projects\bartdesmet-corefx\corefx\Tools\publishtest.targets(97,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
1>C:\Projects\bartdesmet-corefx\corefx\Tools\publishtest.targets(97,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
```

--- a/src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets
+++ b/src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets
@@ -13,7 +13,8 @@
   </PropertyGroup>
 
   <Target Name="CopyTestToTestDirectory"
-          DependsOnTargets="DiscoverTestInputs" Condition="'$(DisableCopyTestToTestDirectory)'!='true'">
+          Condition="'$(DisableCopyTestToTestDirectory)'!='true'"
+          DependsOnTargets="DiscoverTestInputs;GetDefaultTestRid">
 
     <PrereleaseResolveNuGetPackageAssets Condition="Exists($(ProjectLockJson))"
                                AllowFallbackOnTargetSelection="true"

